packages/alfa-style/src/property/display.ts

```diff
@@ -76,54 +83,172 @@ export namespace Specified {
  */
 export type Computed = Specified;
 
+/**
+ * {@link https://drafts.csswg.org/css-display/#typedef-display-outside}
+ */
+const parseDisplayOutside = Keyword.parse("block", "inline", "run-in");
+
+/**
+ * {@link https://drafts.csswg.org/css-display/#typedef-display-inside}
```

Comment:
We have several properties for which we link to CSS Drafts, and other to MDN. Part of the MDN switch seems to trace back to
https://github.com/Siteimprove/alfa/commit/ba1c4743e9c79c9f38988721f079c8911d41dc2e

I'm not sure which is best, but we should stick to one and not mix both.

Reply:
The properties themselves always link to MDN as it provides very broad information aimed at callers. Only the internal bits link to the CSS specifications themselves.